### DISCUSSION TITLE
Overall p-value for model

### DISCUSSION THREAD

**Discussion by Abbie-Williams** [FrederickHuangLin/ANCOMBC#79] [Q&A] [answered]
Hi, 

I was wondering how to find the overall p-value for a model? So, if I have understood this correctly, a GLM is ran for each OTU according to the variables we give. In the results, we see the p-value for the level(s) of that variable compared to the reference level.  Is there a way to see the overall model p-value for each OTU, rather than for individual comparisons?

Thanks, 

Abbie

**Comment by FrederickHuangLin** [comment, marked as answer]
Hi @Abbie-Williams,

If you specify the `group` variable and set `global = TRUE`, the output of `res_global` is what you are looking for.

Best,
Huang
